chart/templates/deployment.yaml

```diff
@@ -36,6 +36,8 @@ spec:
           {{- if .Values.startupConfig}}
           - --config
           - /startupconfig/.sparrow.yaml
+          - --tmconfig 
+          - /startupconfig/tmconfig.yaml
```

Comment:
I think we should refactor this in the future, so we don't have a separate config file for the target manager

Reply:
Yes, this needs to be involved in the config refactoring